server/db/model/Event.js

```diff
@@ -1,4 +1,65 @@
-/* eslint-disable */
 var mongoose = require('../index.js'),
-    config = require('../../../config/default.js'),
-    Schema = mongoose.Schema;
+    Schema = mongoose.Schema,
```

Comment:
we dont use the Schema object here and instead just use `mongoose.Schema` so just remove this (I'm not sure why we had it in the first place